clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -20,6 +20,7 @@ interface InputProps {
   disabled?: boolean;
   label: string;
   tooltip?: string;
+  "data-testid"?: string;
```

Comment:
Being explicit with the test id does seem like a good idea, but I am wondering if having the inputs automatically use `data-testid={`input-${field.name}`} would get us 90% of the way. Form field names are usually unique, so I think it would be relatively robust.